Simplenote/NoteEditorViewController+Swift.swift

```diff
@@ -100,10 +122,47 @@ extension NoteEditorViewController {
         addChild(markdownViewController)
     }
 
-    @objc
-    func dismissMarkdownPreview() {
-        markdownViewController.markdown = nil
+    private func detachMarkdownViewController() {
         markdownViewController.view.removeFromSuperview()
         markdownViewController.removeFromParent()
     }
 }
+
+
+// MARK: - Notifications
+//
+extension NoteEditorViewController {
+
+    @objc
+    func startListeningToScrollNotifications() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(clipViewDidScroll),
+                                               name: NSView.boundsDidChangeNotification,
+                                               object: scrollView.contentView)
+    }
+
+    @objc
+    func clipViewDidScroll(sender: Notification) {
+        refreshTopDividerAlpha()
+    }
+
+    func refreshTopDividerAlpha() {
```

Comment:
Nitpick, but I think this and the following var could be private.

Reply:
**Definitely** private. Thank you!!